db/structs.go

```diff
@@ -460,14 +461,33 @@ type BudgetStoreData struct {
 	Created      *time.Time `json:"created"`
 }
 
+type BudgetPaymentType string
+
+const (
+	Add      BudgetPaymentType = "add"
+	Withdraw BudgetPaymentType = "withdraw"
+)
+
 type BudgetHistory struct {
-	ID           uint       `json:"id"`
-	OrgUuid      string     `json:"org_uuid"`
-	Amount       uint       `json:"amount"`
-	SenderPubKey string     `json:"sender_pubkey"`
-	Created      *time.Time `json:"created"`
-	Updated      *time.Time `json:"updated"`
-	Status       bool       `json:"status"`
+	ID           uint              `json:"id"`
+	OrgUuid      string            `json:"org_uuid"`
+	Amount       uint              `json:"amount"`
+	SenderPubKey string            `json:"sender_pubkey"`
+	Created      *time.Time        `json:"created"`
+	Updated      *time.Time        `json:"updated"`
+	Status       bool              `json:"status"`
+	PaymentType  BudgetPaymentType `json:"payment_type"`
+}
+
+type BudgetHistoryData struct {
+	ID          uint              `json:"id"`
+	OrgUuid     string            `json:"org_uuid"`
+	Amount      uint              `json:"amount"`
+	PaymentType BudgetPaymentType `json:"payment_type"`
+	Status      bool              `json:"status"`
+	SenderName  string            `json:"sender_name"`
+	Created     *time.Time        `json:"created"`
+	Updated     *time.Time        `json:"updated"`
 }
```

Comment:
how is this any different than BudgetHistory?

Reply:
> how is this any different than BudgetHistory?

@kevkevinpal the SenderName column, is the difference, It is not necessary on the budget History table, so I created a new struct for it.